bundles/ch.elexis.core.ui.contacts/src/ch/elexis/core/ui/contacts/views/PatientenListeView.java

```diff
@@ -187,6 +187,7 @@ public void setQuery(IQuery<?> query) {
 									true);
 						} else {
 							query.and(dbFields[i], COMPARATOR.LIKE, lastFiltered[i] + "%", true); //$NON-NLS-1$
+							query.limit(1);
```

Comment:
Wenn hier immer limitiert wird, wird auch für den Filter nach Namen nur 1 Resultat geliefert.

Das dürfte vmtl. auch nur für das Feld code gemacht werden.
<pre>
	if ("dob".equals(dbFields[i])) { //$NON-NLS-1$
		query.and(dbFields[i], COMPARATOR.LIKE, NoPoUtil.getElexisDateSearchString(lastFiltered[i]),
				true);
	} else if ("code".equals(dbFields[i])) {
		query.and(dbFields[i], COMPARATOR.EQUALS, lastFiltered[i]);
	} else {
		query.and(dbFields[i], COMPARATOR.LIKE, lastFiltered[i] + "%", true); //$NON-NLS-1$
	}
</pre>